src/RobotInterface/src/Helper.cpp

```diff
@@ -61,7 +61,6 @@ bool RobotInterface::getFeedbacksRaw(unsigned int maxAttempts)
 
     bool okBaseEstimation = true;
 
-    if(m_useExternalRobotBase)
     okBaseEstimation = !m_useExternalRobotBase;
```

Comment:
You can merge these two lines into a single one:
```c++
bool okBaseEstimation = !m_useExternalRobotBase;
```

Reply:
Done!

Reply:
I still see them separate: https://github.com/robotology/walking-controllers/blob/17c1c7d9fc06d4dfeec31460e250c3447f165a57/src/RobotInterface/src/Helper.cpp#L62-L63

Unfortunately, Github showed in a weird way the lines I was mentioning

Reply:
I think now it should be ok! https://github.com/MiladShafiee/walking-controllers/blob/62fa833e30ad5274be7aa0fd1d8f0cbc8e8c0d63/src/RobotInterface/src/Helper.cpp#L62